apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/name-step/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import '~@automattic/onboarding/styles/variables';
+
+.nux-launch-step__input {
+	position: relative;
+	margin-bottom: 20px;
+
+	input[type='text'].components-text-control__input {
+		padding: 6px 40px 6px 16px;
+		height: 38px;
+		background: #f0f0f0;
```

Comment:
Surprised to see HEX here, instead of variable.

Reply:
👍 Should be the same color as in [domain picker input](https://github.com/Automattic/wp-calypso/blob/master/packages/domain-picker/src/domain-picker/style.scss#L39) according to designs. We should totally use a variable for both.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/44708/commits/569f9356156f735b4d1267e98083c69fca6f8df8